TypeInjections/Tool.Test/ToolTests.fs

```diff
@@ -0,0 +1,148 @@
+namespace Tool.Test
+
+open System
+open System.IO
+open TestUtils
+open Tool
+
+
+module IOTests =
+    open NUnit.Framework
+
+
+
+    [<Test>]
```

Comment:
Please keep the amount of whitespace to a single empty line. Did you run the formatter on this?